### DISCUSSION TITLE
Disabling annotations on chessground (circles & arrows)

### DISCUSSION THREAD

**Discussion by coffeeDev98** [mliebelt/pgn-viewer#387] [Q&A]
Is there any way to disable to on-board annotations on chessground through props passed to chessground?

Came across the `drawable` prop in chessgroundConfig.... can altering this remove the annotations?

**Comment by mliebelt** [comment]
Good question. At the moment, there is no configuration to suppress already existing notations. You were asking about the pgnView configuration, right? 

It would be easy to add something like `{ drawable: false }` that will then suppress the default configuration you have found. Is that the right understanding?

**Reply by coffeeDev98** [reply]
Yes yes. I think this prop can be linked with the Chessground config `drawable` prop directly.... right?

**Reply by coffeeDev98** [reply]
I tried setting it to false within the library like below
<img width="463" alt="image" src="https://user-images.githubusercontent.com/63295908/182014467-a316b80b-fc15-40d4-a89e-0bd79b91e979.png">

But am getting this error when trying to initaialize
<img width="393" alt="image" src="https://user-images.githubusercontent.com/63295908/182014525-cbfcb2e1-f668-445e-8aa9-86f3572f96ab.png">

Where else do you interact with the drawable component in the library? Or is this internal to chess

**Comment by mliebelt** [comment]
I implemented a first version (see [commit](https://github.com/mliebelt/PgnViewerJS/commit/cfb8c36c2148d71eb27944c29dd7dcdac4d4ab05) )

To try that out, do the following:

* You have to clone the repository.
* Then do a full build of the pgn-viewer.
* Start the http-server in directory `modules/pgn-viewer`.
* Go to http:localhost:8080/examples/tickets/ticket387.html

There you have 2 boards (with the same pgn). Only difference is the additional configuration `drawable: false`.

I will later integrate this into a new version, but  I don't know when that will be.
